code-challenges.js

```diff
@@ -41,23 +75,63 @@ var vowelTester1 = "learn"
 var vowelTester2 = "throw"
 // Expected output: 3
 
+//create a function that takes in a string
+const firstVowel = (string) => {
+  //write a for loop that loops through the length of the string
+  for(i=0; i<string.length; i++)
+  //test if the value at each index of the string is a vowel
+  if (string[i] == "a" || string[i] == "e" || string[i] == "i" || string[i] == "o" || string[i] == "u" || string[i] == "A" || string[i] == "E" || string[i] == "I" || string[i] == "O" || string[i] == "U"){
+    //if the value is a vowel then return that index
+    return i
+  }
+}
 
+console.log(firstVowel(vowelTester1))
+console.log(firstVowel(vowelTester2))
 
 
+//   if string.charAt(0) === "a" && string.charAt(0) === "e" && string.charAt(0) === "i" && string.charAt(0) === "o" && string.charAt(0) === "u" && string.charAt(0) === "A" && string.charAt(0) === "E" && string.charAt(0) === "I" && string.charAt(0) === "O" && string.charAt(0) === "U"
+// }
 
 
-// --------------------5) Create a function that takes three arguments - two numbers and a mathematical operation (+, -, *, /) and performs the given calculation. If the input tries to divide by 0, return: "Can't divide by 0!"
 
 
 
 
+// --------------------5) Create a function that takes three arguments - two numbers and a mathematical operation (+, -, *, /) and performs the given calculation. If the input tries to divide by 0, return: "Can't divide by 0!"
+
+//create a function that takes in a number, a mathematical operator, and another number as arguments
+const calculator = (numb1, symbol, numb2) => {
+  //
+  if(symbol === "/" && numb2 === 0){
+    return "Can't divide by 0!"
+  } else if(symbol === "/"){
+    return numb1 / numb2
+  } else if(symbol === "*"){
+    return numb1 * numb2
+  }else if(symbol === "+"){
+    return numb1 + numb2
+  } else if(symbol === "-"){
+    return numb1 - numb2
+  } else if(symbol === "%"){
+    return numb1 % numb2
+  } else {
+    return `${symbol} is not a valid mathematical operator`
+  }
+}
+
+//I initially tried to just use the symbol as a mathematical argument but I couldn't get it to perform the function in the return, so I had to separate it to a bunch of if/else statements. I think there is probably a better or shorter way to do this. 
+
 
 // Uncomment and use the following console logs to test your function
-// console.log(calculator(3, "*", 9))
+console.log(calculator(3, "*", 9))
 // Expected output: 27
 
-// console.log(calculator(16, "+", 3))
+console.log(calculator(16, "+", 3))
 // Expected output: 19
 
-// console.log(calculator(89, "/", 0))
+console.log(calculator(89, "/", 0))
 // Expected output: "Can't divide by 0!"
+
+console.log(calculator(50, "#", 12))
```

Comment:
You did great! I think the key here is to look at the console.logs provided to see how he symbol is being passed in. As we see, its being passed as a string, so that means we need to evaluate that string.  

Look into the built in method, Eval.